rad/monadic/issues.rad

```diff
@@ -56,13 +73,13 @@
 (def create-issue!
   "Create a new remote issue with the keys in `my-keys.rad`."
   (fn [i]
-    (send! (lookup :url (read-ref issues-chain!))
+    (send! (lookup :url (read-ref issues-chain))
```

Comment:
Seems like a lot of `!` are getting lost.

Reply:
These functions are not effectful so I removed the `!`.

Reply:
`issue-chain` *is* a ref, should refs have `!`?

The other functions don't have effects because all they do is read a ref (which I thought we agreed was not an effect), the actual effectful stuff is all concentrated in `fetch!`.

Reply:
OK, so @jameshaydon convinced me on chat that not having a `!` is reasonable.